apps/heft/src/utilities/subprocess/test/SubprocessRunnerBase.test.ts

```diff
@@ -14,6 +15,12 @@ describe('SubprocessRunnerBase', () => {
     const error: Error = new Error();
     error.stack = 'ERROR STACK';
     expect(SubprocessRunnerBase.serializeForIpcMessage(error)).toMatchSnapshot();
+    const fileError1: FileError = new FileError('message', 'path/to/file', 4, 29);
+    fileError1.stack = 'ERROR STACK';
+    expect(SubprocessRunnerBase.serializeForIpcMessage(fileError1)).toMatchSnapshot();
+    const fileError2: FileError = new FileError('message', 'path/to/file', 4);
+    fileError2.stack = 'ERROR STACK';
+    expect(SubprocessRunnerBase.serializeForIpcMessage(fileError2)).toMatchSnapshot();
```

Comment:
Another idea would be to have a unit test that serializes and then deserializes an object, and asserts that the result is equal to the original input.

Reply:
That's what the test case beginning on line 26 is doing.